rotkehlchen/history/events/structures/asset_movement.py

```diff
@@ -31,14 +31,25 @@
     from rotkehlchen.accounting.mixins.event import AccountingEventMixin
     from rotkehlchen.accounting.pot import AccountingPot
     from rotkehlchen.fval import FVal
+    from rotkehlchen.types import Fee
 
 logger = logging.getLogger(__name__)
 log = RotkehlchenLogsAdapter(logger)
 
 
-class AssetMovement(HistoryBaseEntry):
+class AssetMovementExtraData(TypedDict):
+    address: NotRequired[str]
+    transaction_id: NotRequired[str]
+    reference: NotRequired[str]
```

Comment:
Wherever you prefer, either in the docstring or on theright or on top of each with a `#` add a small description of what each field is. Especially useful for reference and transaction id